release-notes/opensearch-release-notes-1.3.2.md

```diff
@@ -0,0 +1,59 @@
+# OpenSearch and OpenSearch Dashboards 1.3.2 Release Notes
+
+## Release Details
+
+OpenSearch and OpenSearch Dashboards 1.3.2 includes the following features, enhancements, bug fixes, infrastructure, documentation, maintenance, and refactoring updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-1.3.2.md).
```

Comment:
Assume this link to be the one for OpenSearch release note after this PR merges. https://github.com/opensearch-project/OpenSearch/pull/3202/files